compair/api/comparison_example.py

```diff
@@ -73,7 +76,9 @@ def delete(self, course_uuid, assignment_uuid, comparison_example_uuid):
         course = Course.get_active_by_uuid_or_404(course_uuid)
         assignment = Assignment.get_active_by_uuid_or_404(assignment_uuid)
         comparison_example = ComparisonExample.get_active_by_uuid_or_404(comparison_example_uuid)
-        require(DELETE, comparison_example)
+        require(DELETE, comparison_example,
+            title="Comparison Example Deletion Failed",
+            message="You do not have permission to delete this comparison example since you are not the course's instructor.")
```

Comment:
"Comparison Example Not Deleted" / "Your role in this course does not allow you to delete comparison examples."